app/views/collections/show.rb

```diff
@@ -102,6 +103,21 @@ def version
 
     private
 
+    def og_image
+      og_image_from_promo || og_image_from_hero
+    end
+
+    def og_image_from_promo
+      return unless @landing_page.settings_layout_type == 'browse'
+      promo = @landing_page.promotions.find_by(position: 0)
+      promo && promo.file.present? ? promo.file.url : nil
+    end
+
+    def og_image_from_hero
+      hero_conf = hero_config(@landing_page.hero_image)
+      URI.join(root_url, hero_conf[:hero_image]) unless hero_conf.nil? || hero_conf[:hero_image].blank?
```

Comment:
Why are we doing `URI.join(root_url, hero_conf[:hero_image])` when the images are stored on S3 in our CF deployments?